src/ResourceManager/Insights/Commands.Insights/Events/GetAzureLogCommand.cs

```diff
@@ -0,0 +1,117 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Insights.OutputClasses;
+
+namespace Microsoft.Azure.Commands.Insights.Events
+{
+    /// <summary>
+    /// Get the list of events for at a subscription level.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureLog"), OutputType(typeof(List<IPSEventData>))]
+    public class GetAzureLogCommand : EventCmdletBase
```

Comment:
All ResourceManager cmdlets should have AzureRm in the name - e.g. Get-AzureRmLog